CondTools/CTPPS/plugins/CTPPSPixGainCalibsESAnalyzer.cc

```diff
@@ -73,9 +75,8 @@ void CTPPSPixGainCalibsESAnalyzer::analyze(const edm::Event& e, const edm::Event
                                               << "\" does not exist ";
   }
   //this part gets the handle of the event source and the record (i.e. the Database)
-  edm::ESHandle<CTPPSPixelGainCalibrations> calhandle;
+  edm::ESHandle<CTPPSPixelGainCalibrations> calhandle = context.getHandle(tokenCalibration_);
   edm::LogPrint("CTPPSPixGainCalibsReader") << "got eshandle";
-  context.get<CTPPSPixelGainCalibrationsRcd>().get(calhandle);
   edm::LogPrint("CTPPSPixGainCalibsReader") << "got context";
   const CTPPSPixelGainCalibrations* pPixelGainCalibrations = calhandle.product();
```

Comment:
For future reference, this could simply have been
```C++
const CTPPSPixelGainCalibrations* pPixelGainCalibrations = &context.getData(tokenCalibration_)
```